src/main/java/com/epam/ta/reportportal/core/widget/util/WidgetOptionUtil.java

```diff
@@ -37,29 +39,31 @@ private WidgetOptionUtil() {
 		//static only
 	}
 
+	@Nullable
 	public static String getValueByKey(String key, WidgetOptions widgetOptions) {
-		expect(widgetOptions, notNull()).verify(ErrorType.OBJECT_RETRIEVAL_ERROR, "Widget options should be not null.");
-		expect(widgetOptions.getOptions(), notNull()).verify(ErrorType.OBJECT_RETRIEVAL_ERROR, "Widget options should be not null.");
-		Object value = widgetOptions.getOptions().get(key);
-		ofNullable(value).ifPresent(v -> expect(v, String.class::isInstance).verify(
+
+		Optional<Object> value = ofNullable(widgetOptions).map(wo -> ofNullable(wo.getOptions()).map(options -> options.get(key))
+				.orElse(null));
+
+		value.ifPresent(v -> expect(v, String.class::isInstance).verify(
 				ErrorType.OBJECT_RETRIEVAL_ERROR,
 				Suppliers.formattedSupplier("Wrong widget option value type for key = '{}'. String expected.", key)
 		));
 
-		return (String) value;
+		return (String) value.orElse(null);
```

Comment:
let's it better be value.get() cause you have validated it

Reply:
decided to leave *.orElse(null) statement